Tutorials/hello-world.md

```diff
@@ -2,9 +2,9 @@
 toc: true
 title: Hello, World!
 tutorial: hello-world
-tutorial-order: 0-hello
-description: an introduction to seL4 projects and tutorials.
+layout: tutorial
 SPDX-License-Identifier: CC-BY-SA-4.0
-SPDX-FileCopyrightText: 2020 seL4 Project a Series of LF Projects, LLC.
+SPDX-FileCopyrightText: 2024 seL4 Project a Series of LF Projects, LLC.
```

Comment:
Please keep all copyright years as they were originally, or if you want to add to it, at least change it to 2020-2024.